### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by jjw24**

### CODE REVIEW

File: Flow.Launcher/Languages/ja.xaml [hunk 1]
```diff
@@ -90,7 +89,7 @@
     <system:String x:Key="author">by</system:String>
     <system:String x:Key="plugin_init_time">初期化時間:</system:String>
     <system:String x:Key="plugin_query_time">クエリ時間:</system:String>
-    <system:String x:Key="plugin_query_version">バージョン</system:String>
+    <system:String x:Key="plugin_query_version"></system:String>
```
Comment (jjw24) [inline Flow.Launcher/Languages/ja.xaml:92]: hm should this be empty?

File: Flow.Launcher/Languages/ja.xaml [hunk 2]
```diff
@@ -253,7 +252,7 @@
     <system:String x:Key="actionkeyword_tips">アクションキーボードを指定しない場合、* を使用してください</system:String>
 
     <!--  Custom Query Hotkey Dialog  -->
-    <system:String x:Key="customeQueryHotkeyTitle" />
+    <system:String x:Key="customeQueryHotkeyTitle"></system:String>
```
Comment (jjw24) [inline Flow.Launcher/Languages/ja.xaml:255]: empty also?

File: Flow.Launcher/Languages/ko.xaml [hunk 3]
```diff
@@ -90,7 +89,7 @@
     <system:String x:Key="author">제작자</system:String>
     <system:String x:Key="plugin_init_time">초기화 시간:</system:String>
     <system:String x:Key="plugin_query_time">쿼리 시간:</system:String>
-    <system:String x:Key="plugin_query_version">버전</system:String>
+    <system:String x:Key="plugin_query_version">버</system:String>
```
Comment (jjw24) [inline Flow.Launcher/Languages/ko.xaml:92]: @onesounds is this correct or is the translation cutoff?

File: Plugins/Flow.Launcher.Plugin.Explorer/Languages/es.xaml [hunk 4]
```diff
@@ -47,9 +47,9 @@
     <system:String x:Key="plugin_explorer_path_enumeration_engine_none">Enumeración directa</system:String>
 
     <system:String x:Key="plugin_explorer_Content_Search_Engine">Motor de búsqueda de contenido</system:String>
-    <system:String x:Key="plugin_explorer_Directory_Recursive_Search_Engine">Motor de búsqueda recursivo de directorio</system:String>
+    <system:String x:Key="plugin_explorer_Directory_Recursive_Search_Engine">Motor de búsqueda recursiva de directorio</system:String>
     <system:String x:Key="plugin_explorer_Index_Search_Engine">Motor de búsqueda del Índice</system:String>
-    <system:String x:Key="plugin_explorer_Open_Window_Index_Option">Abrir ventana de opciones de indexación</system:String>
+    <system:String x:Key="plugin_explorer_Open_Window_Index_Option">Open Windows Index Option</system:String>
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.Explorer/Languages/es.xaml:52]: reverted back to english?

File: Plugins/Flow.Launcher.Plugin.WebSearch/Languages/fr.xaml [hunk 5]
```diff
@@ -30,7 +32,7 @@
 
     <!--  web search edit  -->
     <system:String x:Key="flowlauncher_plugin_websearch_title">Title</system:String>
-    <system:String x:Key="flowlauncher_plugin_websearch_enable">Enable</system:String>
+    <system:String x:Key="flowlauncher_plugin_websearch_enable">Status</system:String>
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.WebSearch/Languages/fr.xaml:35]: Status?

File: Flow.Launcher/Languages/ja.xaml (hunk 1 above)
Comment (taooceros) [reply Flow.Launcher/Languages/ja.xaml:92]: Some source string is changed. So the translated version become null

**Review by taooceros — APPROVED** [review]

**Review by onesounds — APPROVED** [review]

**Pull request merged**